### PR TITLE
Improve cookie management, fix Google Analytics

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
Replaces custom cookie service to universal-cookie. Moves cookie management to React Hooks under `/hooks`.
1. Removes custom cookies service: `/services/cookies.js`
2. Removes `/services` directory since it's empty
3. Creates `useCookies(names)` hook that follows `document.cookie`
   - this obviously doesn't track `document.cookie`

Replaces `next-ga` with `react-ga`
1. Removes `next-ga`
2. Adds `useAnalytics(cookieName)` hook to track the analytics permission and initialises Google Analytics based on the environment:
   - production uses actual Google Analytics
   - development uses `console.log`
   - if disabled, an empty function is used
   - automatic permission tracking obviously doesn't work, see above
3. Adds `usePageviewTracking()` hook that connects to the router and tracks page views
4. Injects all hooks into Next's Custom App
   - adds a workaround because of above

Makes the Analytics permission **optional** but enabled by default.

Updates Cookie page event listeners and functionality with the new hooks. Updates cookie settings config.

To sum up:
1. Fixes the problem that the Analytics is not working
2. Let's accept cookies without refreshing the page

### CODE REVIEW

**Review by Joozty — APPROVED** [review]
I had a brief look and didn't test it. However, it looks good to me. Nice job. 🙂
